tools/data_exploration/lcbd.xml

```diff
@@ -0,0 +1,163 @@
+<tool id="tool_beta" name="Local contributions to beta diversity" version="@VERSION@" profile="20.01">

```

Comment:
the ID should be adopted to the other IDs ... e.g. `ecology_stat_presence_abs`, so maybe `ecology_beta_diversity`.

the qiime tools can not be used here? https://usegalaxy.eu/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fqiime_beta_diversity%2Fqiime_beta_diversity%2F1.9.1.0 and co?

Reply:
The Qiime tool is very genetics / phylogeny oriented so not directly reusable if I am not wrong. Maybe a future plan can be to think about a way to display for example a "beta diversity" Galaxy tool who can be fitted for genetics or occurences data depending on theme or file formats or others... This is clearly something important to investigate (like for others tools as PCA and so on)...